implement "link" command

maff1989 opened the pull request:
This commit includes all required functionality for the "link" command to work on both Telegram and Discord platforms.

maff1989 committed:
implement "link" command

This commit includes all required functionality for the "link" command
to work on both Telegram and Discord platforms.

maff1989 commented:
@mitchellurgero please review the Discord changes. Not sure why you don't show up in my list of reviews to assign.

mitchellurgero reviewed (commented):
Looks good to me! I will be able to test beta in a bit before I approve

maff1989 committed:
bugfix: reverse WalletManager.updateKey params

Had these reversed, causing an undefined error when trying to update
a WalletKey on successful link. This commit fixes that method to
properly pass the data to the method.

mitchellurgero reviewed (approved):
Tested and approved!

maff1989 committed:
bugfix: create account first

A bug was found where a user that doesn't yet exist, and tries to run
the "link" command, will fail. This commit ensures the account exists
first before processing the command.
